packages/aws-cdk/bin/cdk.ts

```diff
@@ -157,6 +164,12 @@ async function parseCommandLineArguments() {
           'which skips CloudFormation and updates the resources directly, ' +
           'and falls back to a full deployment if that is not possible. ' +
           "'true' by default, use --no-hotswap to turn off",
+      })
+      .options('logs', {
+        type: 'boolean',
+        default: true,
+        desc: 'Show CloudWatch log events from hotswapped resources in the terminal. ' +
+          "'true' by default, use --no-logs to turn off",
```

Comment:
The "hotswapped" part is not true anymore, we probably should use the same wording we do in the ReadMe:

```suggestion
        desc: 'Show CloudWatch log events from all resources in the selected Stacks in the terminal. ' +
          "'true' by default, use --no-logs to turn off",
```